.github/actions/shellcheck/action.yml

```diff
@@ -0,0 +1,64 @@
+# This is a reusable action for running ShellCheck,
+# a linter for shell scripts (https://shellcheck.net).
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+# For details on each of these input arguments, see:
+#   ../../../workflow-templates/shellcheck.yml
+# which invokes this action.
```

Comment:
Could we switch to having the inputs doc comments be in this file, with the template pointing to it? Otherwise the doc comments will be copied into each templated action and may get out of date as this file evolves.

Reply:
Done.